src/custom/ecospheres/views/HomeView.vue

```diff
@@ -31,94 +37,335 @@ useHead({
+                  <div class="fr-col-md-3">
+                    <img
+                      src="../assets/document-add.svg"
+                      alt=""
+                      class="illustration fr-mx-auto block"
+                      width="64"
+                      height="64"
+                    />
+                  </div>
+                  <div class="fr-col-md-9">
+                    <h3>Associer les données<br />disponibles</h3>
+                    <p>
+                      Les données existantes satisfont-elles les besoins du cas
+                      d’usage&nbsp;? L’association peut se faire dans sa forme
+                      la plus interopérable via une donnée cataloguée sur
+                      data.gouv ou le renseignement d’un URL.
+                    </p>
+                  </div>
+                </div>
+              </li>
+            </ul>
           </div>
         </div>
-        <div class="fr-mt-10v">
-          <h3>Trouvez un bouquet par thématique</h3>
-          <HomeThemes v-if="config.themes" />
-        </div>
       </div>
     </section>
-    <section v-if="searchConfig.display" class="fr-container fr-py-12v">
-      <h2>Jeux de données</h2>
-      <p>
-        Recherchez parmi les données présentes sur <i>ecologie</i
-        ><strong>.data.gouv</strong><i>.fr</i>.
+    <section class="fr-container fr-py-12v">
+      <h2>Les bouquets à découvrir</h2>
+      <ul class="fr-grid-row fr-grid-row--gutters" role="list">
+        <li
+          v-for="topic in lastTopics"
+          :key="topic.id"
+          class="fr-col-12 fr-col-xl-4"
+        >
+          <BouquetCardHome :bouquet="topic" />
+        </li>
+      </ul>
+      <p class="fr-m-0">
+        <RouterLink
+          :to="{
+            name: 'bouquets'
+          }"
+          class="fr-link fr-icon-arrow-right-line fr-link--icon-right"
+        >
+          Voir les bouquets
+        </RouterLink>
       </p>
-      <div class="fr-grid-row fr-grid-row--gutters">
-        <div class="fr-col-12 fr-col-md-8">
-          <DsfrSearchBar
-            v-model="query"
-            button-text="Rechercher"
-            class="search-bar-input"
-            :label="searchConfig.placeholder"
-            placeholder=""
-            :large="true"
-            @search="doSearch"
-          />
-        </div>
+    </section>
+    <section class="fr-container--fluid faq">
+      <div class="fr-container fr-py-16v faq">
+        <h2>Foire aux questions</h2>
+        <HomeFaq />
       </div>
     </section>
   </div>
 </template>
 
 <style scoped>
-.bouquets {
+.bouquets,
+.faq {
   background-color: #f6f6f6;
 }
 .illustration {
   max-width: 100%;
 }
+.organize-data {
+  background-color: white;
+  padding: 2rem 0;
+
+  li {
+    border-right: 1px solid var(--border-default-grey);
+    padding-right: 1rem;
+  }
+
+  li:last-child {
+    border: none;
+  }
+
+  @media (max-width: 992px) {
+    & {
+      padding: 16px;
```

Comment:
J'avais pas précisé celui-là 😬 

```suggestion
      padding: 1rem;
```